pandas/tests/io/json/test_normalize.py

```diff
@@ -66,6 +66,39 @@ def author_missing_data():
          }]
 
 
+@pytest.fixture
+def data_records_path_with_nested_data():
```

Comment:
If you only plan on using these in once function then just define there; no need for fixture